dataactvalidator/config/sqlrules/sqlRules.csv

```diff
@@ -14,151 +14,151 @@ A11,SpendingAuthorityfromOffsettingCollectionsAmountTotal_CPE= CPE aggregate val
+FABS6,"RecordType field must contain one of the following values: ""1"" or ""2"".","RecordType field must contain one of the following values: ""1"" or ""2"".",n,detached_award,fatal,,fabs6_detached_award_financial_assistance
 FABS7,"URI is a required field for aggregate records (i.e., when RecordType = 1)","URI is a required field for aggregate records (i.e., when RecordType = 1)",n,detached_award,fatal,,fabs7_detached_award_financial_assistance
-FABS8,"FiscalYearAndQuarterCorrection is an optional field, but when provided must follow YYYYQ format, where YYYY is a year and Q represents fiscal quarter and takes value 1, 2, 3, or 4.","FiscalYearAndQuarterCorrection is an optional field, but when provided must follow YYYYQ format, where YYYY is a year and Q represents fiscal quarter and takes value 1, 2, 3, or 4.",n,detached_award,fatal,,fabs8_detached_award_financial_assistance_1
-FABS8,"Provided year must be in current or directly previous fiscal year (as compared to the date of submission).","Provided year must be in current or directly previous fiscal year (as compared to the date of submission).",n,detached_award,fatal,,fabs8_detached_award_financial_assistance_2
-FABS9,"Must contain ""MULTIPLE RECIPIENTS"" for aggregate records (i.e., when RecordType = 1).","Must contain ""MULTIPLE RECIPIENTS"" for aggregate records (i.e., when RecordType = 1).",n,detached_award,fatal,,fabs9_detached_award_financial_assistance
-FABS10,"LegalEntityAddressLine1 is required for non-aggregate records (i.e., when RecordType = 2)","LegalEntityAddressLine1 is required for non-aggregate records (i.e., when RecordType = 2)",n,detached_award,fatal,,fabs10_detached_award_financial_assistance_1
-FABS10,"LegalEntityAddressLine1 must be blank for aggregate records (i.e., when RecordType = 1)","LegalEntityAddressLine1 must be blank for aggregate records (i.e., when RecordType = 1)",n,detached_award,fatal,,fabs10_detached_award_financial_assistance_2
+FABS8.1,"FiscalYearAndQuarterCorrection is an optional field, but when provided must follow YYYYQ format, where YYYY is a year and Q represents fiscal quarter and takes value 1, 2, 3, or 4.","FiscalYearAndQuarterCorrection is an optional field, but when provided must follow YYYYQ format, where YYYY is a year and Q represents fiscal quarter and takes value 1, 2, 3, or 4.",n,detached_award,fatal,,fabs8_detached_award_financial_assistance_1
+FABS8.2,Year provided in FiscalYearQuarterCorrection must be in current or directly previous fiscal year (as compared to the date of submission).,Year provided in FiscalYearQuarterCorrection must be in current or directly previous fiscal year (as compared to the date of submission).,n,detached_award,fatal,,fabs8_detached_award_financial_assistance_2

```

Comment:
Why did we remove the quotes around the text here and in several other places? It isn't hurting anything and it's good practice to have it in case we do decide to add a comma at some point. If anything, they should be added around FABS4 because it's the only FABS rule that doesn't have them.